### DISCUSSION TITLE
Set PV label from PVC label

### DISCUSSION THREAD

**Discussion by lapwat** [longhorn/longhorn#3050] [Q&A] [answered]
In my Helm chart, I create a PVC using longhorn storage class. I would like the associated PV to be backed up everyday.

The idea here is to create a daily recurring job targetting PVs with label `daily`.

How can I set the label `daily` on the PV from the PVC config? Should I create a second Storage Class ?

**Comment by jenting** [comment, marked as answer]
You could consider that this volume uses the same Storage Class, and configure the CRON job as a daily format in StorageClass recurringJobs/recurringJobSelector.
Since we have a different design on the recurring job start from v1.2.0, not sure which Longhorn version you're running.
- https://longhorn.io/docs/1.1.2/snapshots-and-backups/scheduling-backups-and-snapshots/#recurring-jobs-assignment-using-a-storageclass
- https://longhorn.io/docs/1.2.0/snapshots-and-backups/scheduling-backups-and-snapshots/#recurring-jobs-assignment-using-a-storageclass

**Reply by lapwat** [reply]
I am using v1.2.0.

Ok I will then create a dedicated StorageClass longhorn-daily.
